db/seeds.rb

```diff
@@ -54,6 +54,15 @@
   info_url: 'http://www.blackgirlscode.com',
   donation_url: 'http://www.blackgirlscode.com'
 }, {
+  name: 'Hillary Clinton Campaign',
+  description: "Aside from the fact that this country is long overdue for a 
+                female President, Hillary Clinton is one of the most qualified 
+                candidates out there.",
+  reason_to_donate: 'I want to support a female president',
+  slug: 'hillary-clinton',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.